src/stores/token.ts

```diff
@@ -33,18 +34,20 @@ export default class Token extends ContractStore {
   @observable public tokens: IObservableArray<Token>;
 
   private account: Account;
+  private txs: Transactions;
 
   public color: number;
   @observable public name: string;
   @observable public symbol: string;
   @observable public decimals: number;
   @observable public balance?: number;
 
-  constructor(account: Account, address: string, color: number) {
+  constructor(account: Account, txs: Transactions, address: string, color: number) {
     super(tokenAbi, address);
```

Comment:
You need to pass `txs` as a 3rd argument to `super` (and not assign `this.txs` at 50)

Reply:
🙏